source/libraries/ContractDeployer.ts

```diff
@@ -249,7 +256,6 @@ export class ContractDeployer {
             throw new Error("Unable to get address for new categorical market.");
         }
         const createMarketHash = await targetReportingWindow.createMarket.bind({ value: marketCreationFee })(endTime, numOutcomes, numTicks, feePerEthInWei, denominationToken, designatedReporter);
-        waitForTransactionToBeSealed(this.ethjsQuery, createMarketHash);
```

Comment:
We don't need to wait before calling `market.getTypeName`?

Reply:
This call was missing the `await` so it wasn't actually waiting.  This line was effectively a no-op, so I removed it.  We _may_ need to wait here, and I'll be digging into the market creation issue tomorrow to find out.  I removed this because at the moment, it wasn't doing anything but clogging up the intertubes.  😄

Reply:
Haha, oops, good catch!  I was probably sleep-deprived when I added that line. :P